frontend/src/store/objectStore.ts

```diff
@@ -143,9 +158,14 @@ export const useObjectStore = defineStore('object', () => {
 
           // Merge and assign
           state.objects.value = difference.concat(response);
+          // Track all the object IDs that the user would have access to in the table (even if filters are applied)
+          if(!tagset && !metadata) {
```

Comment:
wondering if this needs to check array length = 0. i think the call to fetchObjects from the  filters component always passes in an empty array
eg: `if((!tagset || tagset.length > 0) && (!metadata || metadata.length > 0) {`

Reply:
Changing to `if(!tagset?.length && !metadata?.length)` to be safe